src/OpenLoco/src/Drawing/DrawingContext.h

```diff
@@ -4,10 +4,23 @@
 #include "Graphics/PaletteMap.h"
 #include "Types.hpp"
 #include "Ui/Rect.h"
+#include <OpenLoco/Core/EnumFlags.hpp>
 #include <cstdint>
 
 namespace OpenLoco::Drawing
 {
+    enum class DrawRectInsetFlags : uint8_t
```

Comment:
Should we actually call it DrawRectInsetFlags? Considering its in the Drawing namespace we can probably omit the Draw here at least, it makes quite the long lines.